test/unit/resources/aws_s3_bucket_object_test.rb

```diff
@@ -99,9 +99,17 @@ def test_property_object_acl_public
     end
     refute_empty(public_grants)
   end
+
```

Comment:
Here we have added 2 checks.

1. One to check the value of the resource_id
2. To check whether the resource_id is not nil.